namerd/main/src/main/scala/io/buoyant/namerd/DtabHandler.scala

```diff
@@ -31,60 +31,40 @@ class DtabHandler(
 
   def renderDtab(name: String, dtab: Dtab): Future[Response] = {
     val body = s"""
-          <div class="container-fluid">
             <div class="row">
               <div class="col-lg-6">
                 <h2 class="router-label-title">Namespace "$name"</h2>
               </div>
             </div>
             <div class="delegator">
             </div>
-          </div>
 
           <script id="data" type="application/json">{"namespace": "$name", "dtab": ${
       DelegateApiHandler
         .Codec.writeStr(dtab)
     }}</script>
-          <script data-main="../files/js/main-namerd" src="../files/js/lib/require.js"></script>
+          <script data-main="/files/js/main-namerd" src="/files/js/lib/require.js"></script>
```

Comment:
They pretty much are the same thing. The former is a relative path so we go up a directory to get to the js file whereas the latter is an absolute path. I will revert this since "it ain't broke".